app/services/authn_context_resolver.rb

```diff
@@ -11,15 +11,33 @@ def initialize(user:, service_provider:, vtr:, acr_values:)
   end
 
   def resolve
-    if vtr.present?
-      selected_vtr_parser_result_from_vtr_list
+    result
+  end
+
+  def asserted_ial_acr
+    return if vtr.present?
```

Comment:
we only call this method if vtr values aren't present, so i don't think we need this condition

Reply:
I wanted to be defensive with respect to when this is called. I can remove it as vtr is on the way out.